src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/SpecifyStringComparison.cs

```diff
@@ -102,7 +102,12 @@ protected override void InitializeWorker(CompilationStartAnalysisContext context
                 // plus as additional StringComparison parameter. Default StringComparison may or may not match user's intent,
                 // but it is recommended to explicitly specify it for clarity and readability:
                 // https://learn.microsoft.com/dotnet/standard/base-types/best-practices-strings#recommendations-for-string-usage
-                IEnumerable<IMethodSymbol> methodsWithSameNameAsTargetMethod = targetMethod.ContainingType.GetMembers(targetMethod.Name).OfType<IMethodSymbol>();
+                var methodsWithSameNameAsTargetMethod =
+                    GetAccessibleMethodsWithSameNameAsTargetMethod(
+                        invocationExpression,
+                        targetMethod,
+                        oaContext.Operation.SemanticModel);
```

Comment:
```suggestion
                        oaContext.Operation.SemanticModel!);
```

Reply:
Actually you don't need to pass in `oaContext.Operation.SemanticModel` as a parameter. You can directly access it off  `invocationExpression.SemanticModel`